Scuti/Scripts/UI/Store/Offer Details/OfferShowcasePresenter.cs

```diff
@@ -197,50 +197,36 @@ async void DownloadImages()
         }
 
 
-        async void DownloadLargeImagen(int indexLarge)
+        async void DownloadLargeImage(int indexLarge)
         {
             var downloader = ImageDownloader.New(false);
-            var downloads = new List<Task<Texture2D>>();
-
+            string url = String.Empty;
             List<string> newListUrl = new List<string>(m_Urls);
 
             if (newListUrl.Count > 0)
             {
                 if (indexLarge >= 0)
                 {
-                    if (!String.IsNullOrEmpty(newListUrl[indexLarge]) && newListUrl[indexLarge].IndexOf("shopify") != -1 && newListUrl[indexLarge].LastIndexOf(".") != -1)
-                    {
-                        // if source its shopify
-                        newListUrl[indexLarge] = newListUrl[indexLarge].Insert(newListUrl[indexLarge].LastIndexOf("."), "_1024x1024");
-                        downloads.Add(downloader.Download(newListUrl[indexLarge]));
-                    }
-                    else 
-                        downloads.Add(downloader.Download(newListUrl[indexLarge]));
-                    
+                    url = newListUrl[indexLarge];
                 }
                 else
                 {
-                    downloads.Add(downloader.Download(newListUrl[0]));
```

Comment:
@Jpablogs8 this was the main issue. If indexLarge is -1 which is the case for changing variants, then it would not look for shopify link